run/README.namelist

```diff
@@ -798,6 +798,7 @@ Namelist variables for controlling the adaptive time step option:
                                      = 2, Park and Bretherton shallow cumulus from CAM5 (CESM 1_0_1)
                                      = 3, GRIMS shallow cumulus from YSU group
                                      = 5, Deng shallow cumulus from PSU and WRF-Solar
+                                          This scheme only works with MYJ and MYNN PBL schemes 
```

Comment:
Deng shcu works only with MYJ or MYNN PBL schemes

Reply:
@davegill 
Is this comment resolved now?